### ISSUE TITLE
Debian packaging not installable on Debian

### ISSUE THREAD

**Issue by narcisgarcia** [LibreTime/libretime-debian-packaging#15]
At time of LibreTime 3.0.0~alpha.8-1u3 , I've tried "add-apt-repository ppa:libretime/libretime" as indicated, but no success in Debian GNU/Linux 10 (buster - stable).

PPA only has repository for "xenial" and "bionic" versions of OS (Ubuntu-like), I've tried them anyway, but dependencies aren't satisfiable with Debian.

Please, create a "buster" repository in PPA, to make LibreTime installable in this essential distro.

**Comment by paddatrapper** [comment]
While I would like to support Debian, it seems Launchpad does not support Debian release series - https://launchpad.net/ubuntu/+ppas It requires creating and managing our own APT archive. Which I currently don't have time to do

**Comment by narcisgarcia** [comment]
You can move to a more open build/repositories plataform, such as build.opensuse.org

**Comment by jooola** [comment]
@paddatrapper What do you think of https://packagecloud.io/pricing/ ?
Or https://jfrog.com/pricing/ ?

**Comment by jooola** [comment]
Another idea would be to use Github pages to have our apt packages. And build them using CI. Using something like https://github.com/rpatterson/github-apt-repos

**Comment by paddatrapper** [comment]
My only requirements for an apt server are:

* Support standard Debian tools (`dput` in particular)
* Build supported architectures (the PPA builds for amd64, arm64, armhf, i386) from a `_source.changes` upload
* Has a human between a release being cut and the package being built

Why I want a human in the process is because package building is not a release process, it is effectively a downstream development process. Every new package version requires updating dependencies (if necessary), updating `debian/copyright`, changing build rules as required, patching upstream sources to match what Debian expects, making it Lintian clean (not completely possible with LT at the moment). Then the package is tested on all supported distros before being uploaded to the PPA

**Comment by SwampRabbit** [comment]
@paddatrapper it may not fit exactly what you want (not sure if I mentioned it before) but the OpenSUSE Build Service might be something to look at as it creates a repo for a ton of distros.  Its alright to work with overall.
https://openbuildservice.org/
https://build.opensuse.org/

**Comment by paddatrapper** [comment]
OBS is definitely something to dig into. Can potentially replace the PPA and provide packages for both Debian and Ubuntu. Looks like it supports uploads of `_source.changes` files with a wide variety of build architectures. I'll create a test package of libretime there once I've got alpha.10 packaged

**Comment by SwampRabbit** [comment]
@paddatrapper yep, it can do builds for the major Distro families, MX Linux uses it to just build some stubborn packages that have issues with pbuilder.
It is used to distribute official releases of some major applications for example: Lutris, Palemoon, etc.  If nothing else it helps ensure builds are compatible between the different Distros, releases, and arches.
